bindings/core/src/response.rs

```diff
@@ -157,17 +160,17 @@ pub enum Response {
     Raw(Vec<u8>),
     /// Response for:
     /// - [`GetOutput`](crate::method::ClientMethod::GetOutput)
-    OutputWithMetadataResponse(OutputWithMetadataResponse),
+    OutputWithProof(OutputWithProof),
     /// Response for:
     /// - [`GetOutputMetadata`](crate::method::ClientMethod::GetOutputMetadata)
     OutputMetadata(OutputMetadata),
     /// Response for:
     /// - [`GetOutputWithMetadata`](crate::method::ClientMethod::GetOutputWithMetadata)
-    OutputWithMetadata(OutputWithMetadata),
+    OutputWithMetadataFull(OutputWithMetadataFull),
     /// Response for:
     /// - [`GetOutputs`](crate::method::ClientMethod::GetOutputs)
     /// - [`GetOutputsIgnoreErrors`](crate::method::ClientMethod::GetOutputsIgnoreErrors)
-    Outputs(Vec<OutputWithMetadataResponse>),
+    Outputs(Vec<OutputWithMetadata>),
```

Comment:
Bit weird that getOutput returns a different object than what you get from getOutputs, maybe we should add another one to `getOutputsWithMetadata`? Btw this change also breaks the nodejs binding

Reply:
yes, I know it's breaking the bindings....just wanted to get feedback on it first before changing those :+1:

Reply:
aligned type names with TIP